ex21.py

```diff
@@ -51,12 +54,17 @@ def divide(a, b):
 
 # assigning the variable "what" to the return of....
 # fuck I wrote all this backwards ><
-# calling the "divide()" function using 2 and 3 as its arguments, a and b. Then 
+# calling the "divide()" function using 2 and 3 as its arguments, a and b. Then
 # calling the "multiply()" function on the value returned by "divide()" as b and 400 as a,
 # then calling the "add()" function on the returned value of "multiply()" as b and 35 as a,
 # finally calling the "subtract()" function using the returned value of the "add()" function
-# as b and 150 as a 
+# as b and 150 as a
 what = subtract(150, add(35, multiply(400, divide(2, 3))))

```

Comment:
As it happens, you described it as:
a = 2
b = 3
150 - (35 + (400 * (a / b)))

which is correct :)